actions-runner-controller.yaml

```diff
@@ -64,10 +65,27 @@ subpackages:
       - runs: |
           mkdir -p "${{targets.subpkgdir}}"
           ln -sf /usr/bin/manager ${{targets.subpkgdir}}/manager
+          ln -sf /usr/bin/ghalistener ${{targets.subpkgdir}}/ghalistener
 
 update:
   enabled: true
   github:
     identifier: actions/actions-runner-controller
     strip-prefix: gha-runner-scale-set-
     tag-filter: gha-runner-scale-set-
+
+test:
+  environment:
+    contents:
+      packages:
+        - ${{package.name}}-compat
+  pipeline:
+    - runs: |
+        set +e
+        /manager -h
+        /ghalistener -h
+        github-runnerscaleset-listener -h
+        ghalistener -h
+        github-webhook-server -h
+        actions-metrics-server -h
+        sleep 30 &
```

Comment:
I see why this is called sleep now (binary name from above). but can we pass this in as 1 so not to waste CI time / resources?